sdk/storage/azure-storage-blob/migrationGuides/V8_V12.md

```diff
@@ -306,3 +306,7 @@ final BlockBlobClient blobClient = new BlobClientBuilder()
 OutputStream blobOutputStream = blobClient.getBlobOutputStream();
 // insert your method of choice to write to an OutputStream
 ```
+
+## Miscellaneous
+- Examples for how to approximate v8 StorageEvent behavior can be found in samples/StorageEventExample.java
+- Examples for how to approximate v8 LocationMode behavior can be found in samples/LocationModeExample.java
```

Comment:
Could these statements be made into links so that users can jump to them directly